modules/plot/test/spec.coffee

```diff
@@ -91,6 +95,37 @@ describe "plot", ->
       axis.x.scaleType().should.equal('discrete')
       axis.y.scaleType().should.equal('time')
 
+    it 'should not redraw on resize when told not to', ->
```

Comment:
Might it be worth testing the other way around too ? Ie check it does redraw on resize by default.

Mainly as a precaution against errors in this very test.